dts/arm/nxp/nxp_imx95_m7.dtsi

```diff
@@ -546,7 +546,7 @@
 				compatible = "nxp,irqsteer-master";
 				reg = <0>;
 				interrupt-controller;
-				#interrupt-cells = <1>;
```

Comment:
There should be no build issue because you break bisectability. You should put this in the same commit that updates the interrupt-cells and mention this. 

(We are adding all the code here to avoid breaking the build). Remember, a patch should do one logical change per patch.